bg_atlasgen/atlas_scripts/kim_developmental_ccf_mouse.py

```diff
@@ -0,0 +1,341 @@
+__version__ = "1"
+
+import json
+import time
+import tarfile
+
+import pandas as pd
+import numpy as np
+import multiprocessing as mp
+
+from rich.progress import track
+from pathlib import Path
+from scipy.ndimage import zoom
+
+from bg_atlasapi import utils
+from bg_atlasgen.mesh_utils import create_region_mesh, Region
+from bg_atlasgen.wrapup import wrapup_atlas_from_data
+from bg_atlasapi.structure_tree_util import get_structures_tree
+import imio
+import zipfile
+import os
+
+PARALLEL = False  # disable parallel mesh extraction for easier debugging
+
+
+def clean_up_df_entries(df):
+    """
+    Remove ' from string entries in the csv
+    """
+    df["Acronym"] = df["Acronym"].apply(lambda x: x.replace("'", ""))
+    df["Name"] = df["Name"].apply(lambda x: x.replace("'", ""))
+    df["ID"] = df["ID"].apply(
+        lambda x: int(x)
+    )  # convert from numpy to int() for dumping as json
+
+    ints = [int(ele) for ele in df["ID"]]
+    df["ID"] = ints
+
+
+def get_structure_id_path_from_id(id, id_dict, root_id):
+    """
+    Create the structure_id_path for a region
+    from a dict mapping id to parent_id
+    """
+    structure_id_path = [id]
+    if id == root_id:
+        return structure_id_path
+
+    while True:
+
+        parent = int(id_dict[id])
+        structure_id_path.insert(0, parent)
+
+        if parent == root_id:
+            break
+
+        id = parent
+
+    return structure_id_path
+
+
+def create_atlas(working_dir, resolution, reference_key, reference_filename, mesh_creation, existing_mesh_dir_path=None):
+    """"""
+    ATLAS_NAME = f"kim_developmental_ccf_mouse_{reference_key}"
```

Comment:
Could these names be shortened? These names have to be typed when using the atlases, and it's easy to make mistakes with something like:

`brainreg input output -v 10 10 10 --orientation psl --atlas kim_developmental_ccf_mouse_lsfm_idisco_10um`

Reply:
`kim_dev_mouse_idisco_10um` and `kim_dev_mouse_standard_10um` sound good. Although instead of `standard` can we be more specific? Is it a serial 2p reference image?

Reply:
It is the "CCFv3_average_template", I see the CCFv3 is a 2p Allen template (1675 brains!) (original filename is "CCFv3_average_template_ASL_Oriented_u16_10um.nii").  In case that ```kim_dev_mouse_CCFv3_10um```?

Reply:
I'd go for `kim_dev_mouse_STP_10um` or similar. CCFv3 refers more to the coordinate space, which is the same for all the atlases.